airbyte-ci/connectors/metadata_service/lib/tests/test_gcs_upload.py

```diff
@@ -12,6 +12,44 @@
 from metadata_service.constants import METADATA_FILE_NAME
 
 
+def stub_is_image_on_docker_hub(image_name: str, version: str) -> bool:
+    return "exists" in image_name and "exists" in version
+
+
+def setup_upload_mocks(mocker, version_blob_md5_hash, latest_blob_md5_hash, local_file_md5_hash):
```

Comment:
nit: in pytest the `setup` `teardown` pattern is usually declared through the use of fixtures.